api/control/v1alpha1/types.go

```diff
@@ -366,9 +356,13 @@ type BindingList struct {
 	Items           []Binding `json:"items"`
 }
 
-// StatusCollector defines one way to collect status about a given workload object from
+// StatusCollector defines a way to collect and report status about a given workload object from
 // the set of WECs that it propagates to.
-// This is modeled after an SQL SELECT statement that does aggregation.
+// There are two styles of StatusCollectors, one general and one specialized.
+// The general style is modeled after a simple SQL SELECT statement and may or may not do aggregation;
+// the corresponding results go in CombinedStatus objects.
+// The specialized style is meant for the case where propagation is to exactly 1 WEC and picks out
+// parts of the object state to propagate back from that WEC to the WDS.
```

Comment:
Could the status subresource contain fields that must be set once status is not empty?